.circleci/config.yml

```diff
@@ -1478,123 +2822,309 @@ jobs:
 ##############################################################################
 ##############################################################################
 
+# PR jobs pr builds
 workflows:
   version: 2
   build:
     jobs:
-      - pytorch_linux_trusty_py2_7_9_build
-      - pytorch_linux_trusty_py2_7_9_test:
-          requires:
-            - pytorch_linux_trusty_py2_7_9_build
-      - pytorch_linux_trusty_py2_7_build
-      - pytorch_linux_trusty_py2_7_test:
+      - binary_linux_manywheel_2.7mu_cpu_build
```

Comment:
To access the `org-member` variables, we should likely put `context: org-member` under this item (at the same indentation level as `requires:`.

Reply:
This worked. I wonder why I can't specify this at the job definition level